searx/settings_loader.py

```diff
@@ -30,29 +31,29 @@ def load_yaml(file_name):
 
 
 def get_default_settings_path():
-    return check_settings_yml(join(searx_dir, 'settings.yml'))
+    return existing_filename_or_none(join(searx_dir, 'settings.yml'))
 
 
-def get_user_settings_path():
-    # find location of settings.yml
+def get_user_settings_path() -> Optional[str]:
+    """Get an user settings file.
+    By descending priority:
+    1. ``environ['SEARXNG_SETTINGS_PATH']``
+    2. ``/etc/searxng/settings.yml`` except if ``SEARXNG_DISABLE_ETC_SETTINGS`` is ``true`` or ``1``
+    3. ``None``
+    """
```

Comment:
For the future:  reST is a plain text markup. Plain text markups are made to be readable in plain text and with a pinch of salt and pepper it can be parsed and rendered into another format ... 

Empty lines are used to introduce the next block (paragraph, list, ..). A single newline does not introduce a new block, it just exists to wrap the line in the plain text.

```
Get an user settings file.
By descending priority: ..
```

is rendered in one paragraph / it is the same like 

```
Get an user settings file. By descending priority: ..
```

The (numbered) list is a new block, to help the parser its recommended to add a empty line that introduce the new block (the list):

```
Get an user settings file. By descending priority:

1. foo
2. bar
```